mlflow/cli.py

```diff
@@ -195,7 +194,6 @@ def server(file_store, default_artifact_root, host, port, workers, static_prefix
         sys.exit(1)
 
 
-cli.add_command(mlflow.sklearn.commands)
```

Comment:
This is technically a breaking change, but I guess it's fine since the new one seems to be a strict superset of the functionality.